pytest-embedded-wokwi/README.md

```diff
@@ -35,3 +35,17 @@ Windows PowerShell:
 ```powershell
 $env:WOKWI_CLI_TOKEN="your-api-key"
 ```
+
+#### Usage
+
+To run your tests with Wokwi, make sure to specify the `wokwi` service when running pytest, e.g.:
+
+```
+pytest --embedded-services idf,wokwi
+```
+
+If your tests run for longer than 30 seconds, you may have to increase the simulation timeout. For example, to set the timeout to 60 seconds (60000 milliseconds):
+
+```
+pytest --embedded-services idf,wokwi --wokwi-timeout=60000
```

Comment:
Shouldn't this somehow be derived from the timeouts specified in dut.expect(...) calls? Otherwise, we have to specify the timeout in two places: in dut.expect and as a CLI argument.

Another option, can we simply let the simulation run without a timeout? Timeout in dut.expect will still interrupt the simulation if the expressions isn't matched, which will fail the test and stop the simulation.

Reply:
Thanks for the feedback!

Since the `dut.expect(...)` calls are only evaluated at runtime, we can't know in advance how long the timeout should be.

The purpose of the timeout is to safeguard against errors (such as a case where the test gets stuck in an infinite loop). The simulation is computationally expensive, so we wouldn't want this process to keep running for hours in case of such error. We can bump the default from 30 seconds to a larger value, but from my experience so far, 30 seconds is enough for more test cases (that's why it's also the current default).

Reply:
Right, but why would the simulation be running for such a long time without the pytest test case exiting?

If the test case calls `dut.expect(...)` and the simulation doesn't output the expected expression, the test case will fail and the simulation will get stopped.

Reply:
Normally it wouldn't. For wokwi-cli (in general), it makes sense to have a time out since it's very easy to just run it in a terminal and forget it running. 

In the specific case of pytest embedded, I think only some sort of developer error could cause it to run for a longer time (e.g. users trying to implement their own expect mechanism instead of using `dut.expect()`. 

At some point we plan to start billing for CI usage, so, what I'm essentially worried about is the user experience - if a developer had such error and the CI keeps running for hours while it would normally complete within seconds, it could become rather expensive for the user.

Reply:
@urish I totally agree that setting a default timeout could be helpful for developers cutting unneeded expenses. However, I'd like to propose adding a complementary flag, `--long-lasting`, to explicitly disable the timeout for certain use cases.

This flag will mostly benefit users who integrate wokwi-cli into CI platforms like GitHub Actions or GitLab. These platforms often have their own timeout settings, and providing the `--long-lasting` flag would allow users to configure the timeout centrally while still respecting Wokwi's cost considerations.

I believe this approach balances both cost concerns and user flexibility. WDYT?

Reply:
Let's see if I get it right, the suggestion is to add a `--long-lasting` flag to the Wokwi CLI, and then use it always when running from within pytest-embedded? Or do we still want to keep the `--wokwi-timeout` flag for pytest and use `--long-lasting` as the default if no timeout provided?

Reply:
I'd prefer the latter one, keep the `--wokwi-timeout` flag and use `--long-lasting` as the default if no timeout specified. Thanks!

Reply:
Alright, I pushed a change where I kept `--wokwi-timeout`, and set the default to one day (which is in essence the same as `--long-lasting`). I also updated the documentation to clarify that this new flag is limiting the simulation time.

Reply:
sure. thanks for the update!